pkg/parcacol/normalizer_test.go

```diff
@@ -21,55 +21,126 @@ import (
 	pprofpb "github.com/parca-dev/parca/gen/proto/go/google/pprof"
 )
 
+func TestLabelNamesFromSamples(t *testing.T) {
+	cases := []struct {
+		name         string
+		takenLabels  map[string]string
+		stringTable  []string
+		samples      []*pprofpb.Sample
+		allLabels    map[string]struct{}
+		allNumLabels map[string]struct{}
+	}{
+		{
+			name: "colliding labels in descending order",
+			takenLabels: map[string]string{
+				"instance": "127.0.0.1:6060",
+			},
+			stringTable: []string{"", "instance", "17", "method", "GET"},
+			samples: []*pprofpb.Sample{
+				{
+					Label: []*pprofpb.Label{{
+						Key: 3,
```

Comment:
@brancz I just want to make sure I get this right 😅 I added some missing tests based on the ones that were already here. The `order` in `colliding labels in descending order` refers to the order of the `[]*pprofpb.Label` right? So the `label.Key` in descending order is `method, instance` in this case and reversed in the ascending case.